### PR TITLE
LPS-70258 Rebased

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ organizations-types-help=Set this JSON object to change the organizations types.
 use-initials-for-default-user-portrait=Use Initials For Default User Portrait
```
Comment (liferay-appsec) [inline modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**